src/components/NavBar/NavIcon.css.ts

```diff
@@ -11,7 +11,6 @@ export const navIcon = style([
     justifyContent: 'center',
     textAlign: 'center',
     cursor: 'pointer',
-    padding: '10',
     borderRadius: '8',
     transition: '250',
```

Comment:
add `alignItems: 'center'` to correctly center the content and you should be good